Added 'audit' zsh script

Phil-Venter opened the pull request:
I am enjoying viskit, helps me clean the codebase... I couldn't find a way to 'audit' the project as a whole, so I wrote a zsh script to call most of the informative viskit commands.

Phil-Venter committed:
A zsh script to 'audit' the project

Phil-Venter committed:
Update audit.zsh

mig82 commented:
Hi @Phil-Venter. I'm glad you like it. I've just added a couple more commands to it. I like your idea. Would you perhaps consider updating your PR to use those commands as well before I merge it?

Philip-Bidvest-Mobility committed:
massive update to audit.zsh

Added support for theme and locale so  more commands (like find-skins, find-dupe-skins, find-font-references, find-dupe-i18ns) can be run as well

Phil-Venter commented:
I think the zsh script includes most of the commands now

find-actions
find-views
find-widgets
find-orphan-widgets
find-autogrow-widgets
find-redundant-containers
find-skins
count-skin-uses
find-dupe-skins
find-font-references
find-dupe-i18ns
find-images

let me know if I'm missing one you think should be included

mig82 commented:
Hi Phil. Looking at this again to merge it. I'm confused as to how you're using the script. See, the idea is to have Viskit installed globally `npm install -g viskit` and then to be able to run it on any project anywhere in the file system —e.g.  like `viskit fv path/to/project`. How are you running the script from anywhere? Are you adding it to your `PATH`?

Phil-Venter commented:
I did not, I created an alias in my `.zshrc` file.
